### ISSUE TITLE
Carousel indicators without ng-repeat

### ISSUE THREAD

**Issue by moisespio** [revolunet/angular-carousel#264]
Hello folks!

I'm not using ng-repeat on my carousel, but I want to use the indicators.
There is any way to do this?

Thanks!

**Comment by moisespio** [comment]
Hello @revolunet 
Any update about this issue?

Thanks!

**Comment by seadour** [comment]
I'll add to the chorus with a "Me too!" -- I can't get this to work either.

Still a great plugin though, even without this capability. Thank you!

**Comment by bornytm** [comment]
+1 in the same boat

**Comment by dirkgroenen** [comment]
Just had the same 'problem' but should be easy to fix by assigning an array to the slides attribute. My code:

```html
<ul class="slidewrap" rn-carousel rn-carousel-index="carouselIndex">
    <li class="slide">
        ...
    </li>
    <li class="slide">
        ...
    </li>
    <li class="slide">
        ...
    </li>
    <li class="slide">
        ...
    </li>
</ul>
<div rn-carousel-indicators slides="[0,1,2,3]" rn-carousel-index="carouselIndex"></div>
```

**Comment by jchaney01** [comment]
I fixed this issue by changing https://github.com/revolunet/angular-carousel/blob/0.3.10/dist/angular-carousel.js#L66 with `indexModel.assign(scope.$parent, index);`

**Comment by jchaney01** [comment]
Just an update, if the `ng-if="slides.length > 1"` is not present, it will not work.  Otherwise, remove the $parent.

**Comment by bornytm** [comment]
@dirkgroenen Defining the array in the html makes the indicators visible, but does clicking on the indicator to switch to the relevant slide work for you?

**Comment by dirkgroenen** [comment]
Nope, unfortunately that's the only thing which doesn't work. But I haven't tried anything to make it work since I don't need it for the current project.

**Comment by calculatorZ** [comment]
I use this method for custom indicators.Hope it helps.

    /*in controller.js*/
    .controller('carousel',['$scope',function($scope){
        $scope.slides = [2,1,0];
    }]);

    /*in html*/
    <div ng-controller="carousel">
        <ul class="featured" rn-carousel rn-carousel-auto-slide rn-carousel-index="carouselIndex">
            <li>First Slide</li>
            <li>Second Slide</li>
            <li>Third Slide</li>
        </ul>

        <div class='indicators'>
            <span class='default' ng-repeat="index in slides" ng-class="{'active':carouselIndex==index}" ></span>
        </div>
    </div>

**Comment by howtomake** [comment]
**EggFiredRice**, thanx, this works )

**Comment by khellhake** [comment]
This thread helped out helped out a lot!  declaring the carouselIndex did the trick for me.  Thanks!
